src/main/java/backend/Logic.java

```diff
@@ -127,9 +127,9 @@ public void refresh() {
         });
     }
 
-    public CompletableFuture<Boolean> getIssueMetadata(String repoId, List<Integer> issues) {
+    public CompletableFuture<Boolean> getIssueMetadata(String repoId, Map<Integer, String> issues) {
```

Comment:
The copy would probably be affected if we're concerned about ETags (which we are), so if the `TurboIssue`s should be copied, then there should be a deeper copy mechanism. Why don't you want the map, though? Too verbose?